### ISSUE TITLE
[Happy Path] Sometimes the "Happy Path" test fails on "ProjectTree.expandPathAndOpenFile" step

### ISSUE THREAD

**Issue by Ohrimenko1988** [eclipse-che/che#14197]
### Describe the bug
Sometimes the "Happy Path" test fails on "ProjectTree.expandPathAndOpenFile" step

### Screenshots
![image](https://user-images.githubusercontent.com/22724619/62854336-bb7d3600-bcf7-11e9-9576-89fd3d379f95.png)

### Console log

[log.txt](https://github.com/eclipse/che/files/3491589/log.txt)

**Comment by Ohrimenko1988** [comment]
Done https://github.com/eclipse/che/pull/14199

**Comment by Katka92** [comment]
Hi guys, from Thursday 22nd tests running against prod-preview Rh-Che with version `7.0.0` started to fail on this issue.

Job console log: https://ci.centos.org/job/devtools-rh-che-periodic-prod-preview-2a/1251/console
Screen of a output that shows that workaround does not work:
![Screenshot from 2019-08-26 13-14-12](https://user-images.githubusercontent.com/29278800/63689419-31aa8e00-c80a-11e9-8593-0b52e5b8b09e.png)

Screenshot from a test:
![Screenshot from 2019-08-26 14-04-22](https://user-images.githubusercontent.com/29278800/63689503-70404880-c80a-11e9-82d0-9ffa5f44cd3b.png)

**Comment by Ohrimenko1988** [comment]
@Katka92 Could you share execution information (logs, dumps, screencasts ...)?

**Comment by Katka92** [comment]
@Ohrimenko1988 I've shared log in my comment, sharing again: : https://ci.centos.org/job/devtools-rh-che-periodic-prod-preview-2a/1251/console
all artefacts we have are here: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-periodic-prod-preview-2a/1251/

**Comment by Katka92** [comment]
@Ohrimenko1988 
Failed again on prod-preview Rh-Che version 7.0.0. 
logs: https://ci.centos.org/job/devtools-rh-che-periodic-prod-preview-2a/1289/console
artefacts: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-periodic-prod-preview-2a/1289/

**Comment by Katka92** [comment]
Today the tests fails again. @Ohrimenko1988 do you plan to spend some time on that? Are you having those issues in your tests too?

**Comment by Ohrimenko1988** [comment]
@Katka92 I guess I have found a reason. According to this message ```"The '/console-java-simple/src/main' item has not been expanded, try again
"```
the test successfully opened ```"src"``` folder and try to open the ```"main"``` folder. But on screenshot we can see that ```"src"``` folder is closed. So it means that something happened and this folder has been closed. I suppose that this "something"  is some build or import project. On the screencast we can see the ```"bin"``` folder which appeared after starting project tree expanding. So it may leeds to the project tree collapsing. At least I suppose so. Try explicitely wait until the ```"bin"``` folder is appeared and just after that expand tree.
